.github/workflows/tests.yml

```diff
@@ -16,7 +16,8 @@ jobs:
     strategy:
       matrix:
         php-version: ['7.4', '8.0']
-        mqtt-broker: ['mosquitto', 'hivemq', 'emqx']
+        mqtt-broker: ['mosquitto']
+#        mqtt-broker: ['mosquitto', 'hivemq', 'emqx']
 
```

Comment:
Was this intended?

Reply:
Yes, I explained it in the commit. If any of the tests fail, GitHub aborts all jobs. And since I'm first trying to implement Mosquitto and not all at once, I want honest feedback for the Mosquitto jobs. But unfortunately, something doesn't work with the volume mount for the Mosquitto config. After having tried changing the owner of the files, mounting not a file but a directory, mounting an entirely different directory and using a read-only mount, it feels like a bug of GitHub or the docker image I'm using as base... not sure yet.

Reply:
Yeah, that stuff is broken. Somewhere, caching is happening and I don't like it a single bit. 😢

Reply:
Wow, problem solved... some stupid caching with environment variables. And who knew you'd need to mount outside directories inside a container into a nested container... since it's actually a sibling container and not a nested container. Gonna clean this up tomorrow then.